README.md

```diff
@@ -32,269 +34,51 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 
 ## Run Deephaven
 
-This section is a quick start guide for running Deephaven from pre-built images.  Almost all users will want to run Deephaven using pre-built images.  It is the easiest way to deploy.  For detailed instructions, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/quickstart).
+If you want to use Deephaven, see our [Quick start for Docker guide](https://deephaven.io/core/docs/tutorials/quickstart).
```

Comment:
We should also be directing people to https://deephaven.io/core/docs/tutorials/quickstart-pip/.  We need to provide very obvious flight paths for docker vs pip users.  This document is so lean that there is one flight path for end users.